main.py

```diff
@@ -1,2 +1,279 @@
-from selenium import webdriver
-driver = webdriver.Chrome()
\ No newline at end of file
+#VARIABLESVARIABLESVARIABLESVARIABLESVARIABLESVARIABLESVARIABLESVARIABLESVARIABLESVARIABLES
+# 1.variables
+a = "12"
+b = 13
+c = 14.0
+print(a is b) #false because is not equal
+print(b is c) #false because is not equal
+print(b is c) #false because is not equal
```

Comment:
Find out what's is the difference between 'is' and '==' operators and compare variables using '==' operator. 
Add comments to the code with explanation